src/styles/all-styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: (4 * $card-width) + (3 * $card-margin-right) + (2 * $padding-horizont);
+  padding: $padding-vertical $padding-horizont;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: $card-width;
+  background-color: $white-color;
+  border: 1px solid $like-white-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-right: $card-margin-right;
+  margin-bottom: $card-margin-bottom;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.